### PR TITLE
Feature/optional dpr param

### PR DESCRIPTION
**Pull request by barneyfoxuk**
This adds to things:

 * Ability to not pass dpr as a param on images as it breaks contentful.  If set to false width is calculated by multiplying by dpr instead.

 * pageTurn event now also passes the index of the page being turned to

### CODE REVIEW

File: src/js/monkey.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ window.Monkey = module.exports = (function () {
     }
 
     promise = promise
-      .then(Monkey._generateUrls(options.preload))
+      .then(Monkey._generateUrls(options.preload, options))
```
Comment (callumacrae) [inline src/js/monkey.js]: just pass in options, `options.preload` is now redundant

Comment (barneyfoxuk) [inline src/js/monkey.js]: Isn't it being used here?

https://github.com/Lostmyname/monkey/blob/master/src/js/steps/generateUrls.js#L49

Comment (callumacrae) [inline src/js/monkey.js]: Yes, but now you have the `options` object so you can just replace `preload` with `options.preload` :)

Comment (barneyfoxuk) [inline src/js/monkey.js]: Gotcha.  Updated.

File: src/js/steps/generateUrls.js [hunk 2]
```diff
@@ -28,7 +29,14 @@ module.exports = function (preload) {
...
 
-    var queryString = 'w=' + size + '&dpr=' + dpr + '&q=' + quality[data.monkeyType];
+    if (!options.dprSupported) {
+      width = 'w=' + Math.round(size * dpr);
+    } else {
+      width = 'w=' + size + '&dpr=' + dpr;
+    }
+
+    var queryString = width + '&q=' + quality[data.monkeyType];
+    // + '&dpr=' + dpr
```
Comment (callumacrae) [inline src/js/steps/generateUrls.js]: what is this?

**Pull request merged**